advanced/list-of-log-names.md

```diff
@@ -1,72 +1,76 @@
 ---
-description: In this page all the available logs are listed
+description: On this page all the available logs are listed
 ---
 
-# List of log names
+# [List of log names](list-of-log-names.md#list-of-log-names)
 
-## Categories
+Logs, categorized by what they do are listed here, and are **case sensitive**. Individual logs are written in _UPPER\_CASE_, while category names (which can be used in place of individual log names, see [_**configuring channels**_](../management-commands/config.md)) are _Capitalized_. Each one of them is separated by categories in the lists below. If you want to log everything, use an asterisk to include all things that can be logged, including ones that will be introduced in future updates. All logs require the bot to have permission to view and send messages in the logging channel. It must also be able to send embeds and files there.
```

Comment:
The anchor to the channels part goes here after config.md, remove the markdown. I realized now how disconnected the sentence is with the linking to guide part is popping off in the middle.